src/components/Missions/Missions.js

```diff
@@ -15,21 +24,29 @@ const Mission = () => {
 
   return (
     <table>
-      <tr>
-        <th>Mission</th>
-        <th>Descriptions</th>
-        <th>Status</th>
-        <th>{String.fromCharCode(8194)}</th>
-      </tr>
-      {missionData.map((data) => (
-        <MissionsItem
-          key={data.mission_id}
-          id={data.mission_id}
-          name={data.mission_name}
-          description={data.description}
-        />
-      ))}
+      <thead>
+        <tr>
+          <th>Mission</th>
+          <th>Descriptions</th>
+          <th>Status</th>
+          <th>{String.fromCharCode(8194)}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {missionData.map((data) => (
+          <MissionsPage
+            key={data.mission_id}
+            joinMissionHandler={joinMissionHandler}
+            leaveMissionHandler={leaveMissionHandler}
```

Comment:
This is optional since we are using redux, you don't need to pass this event handler dispatchers as props, you can call these functions inside the component and access to the state, it will depend on your component design. but the idea of redux is to avoid passing information to the child components and have access to the state without using props. this app is very small and works but for bigger projects may have to pass this info through 10 components and it's going to become a hell manage passing through props